src/main/scala/com/fulcrumgenomics/util/GeneAnnotations.scala

```diff
@@ -25,32 +25,92 @@
 
 package com.fulcrumgenomics.util
 
-import htsjdk.samtools.util.{CoordMath, Interval}
+import htsjdk.samtools.util.{CoordMath, Locatable}
+import com.fulcrumgenomics.FgBioDef._
 
 /** Stores classes useful for storing annotation information for genes and their transcripts and exons. */
 object GeneAnnotations {
+  /**
+    * A gene with a collection of gene loci.
+    */
+  case class Gene(name: String, loci: Seq[GeneLocus]) extends Iterable[Transcript] {
+    def iterator: Iterator[Transcript] = this.loci.iterator.flatten
```

Comment:
this is cute, and since I read this first, I didn't understand that `GeneLocus` implemented `Iterable`.  Perhaps a comment is useful for any future person reading the code?